app/pb_kits/playbook/pb_person_contact/_person_contact.html.erb

```diff
@@ -1,25 +1,26 @@
 <%= content_tag(:div,
```

Comment:
What do you mean by `This in no way is a backward breaking change to the Nitro repo, as it isn't implemented there yet.`? 

I do see this usage:
```
            <%= pb_rails("contact", props: {
              contact_type: "cell",
              contact_value: history.phone_number,
            }) %>
```
this is not using the changed param (wrong number), but I think that until we release this kit someone may use a  `contact_type = "wrong number"`. And that would configure a breaking change. Right? Or I am missing something?

Reply:
The case statement from Line 21 in the `contact.rb` file states which options are available as `contact_type`s. If someone uses anything not currently on the list, the kit will default to the phone icon. So if they use `contact_type = "wrong number"` incorrectly, they'll just get a phone icon with no slash through it.

Reply:
Ok, cool. So this is changing the API but you're handling the exception. Good call.